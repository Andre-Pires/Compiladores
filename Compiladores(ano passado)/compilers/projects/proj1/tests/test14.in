#include <stdio.h>

int x;

/* successor */
/* successor */
int s(int x){
  return x + 1;
}
