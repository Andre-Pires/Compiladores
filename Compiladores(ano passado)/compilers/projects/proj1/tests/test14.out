3 271 int
3 257 x
3 59 ;
7 271 int
7 257 s
7 40 (
7 271 int
7 257 x
7 41 )
7 123 {
8 278 return
8 257 x
8 43 +
8 258 1
8 59 ;
9 125 }
#Total Number of Tokens Matched was 16
